Azure/sonic-buildimage
# Convert docker-database to buster

Signed-off-by: Joyas Joseph <joyas_joseph@dell.com>

**- Why I did it**
This is an enhancement.

**- How I did it**
Code change. 

**- How to verify it**
BLDENV=buster make -f Makefile.work target/docker-database.gz

Check that /etc/apt/sources.list in database container has reference to buster

**- Description for the changelog**
Convert docker-database to buster


**- A picture of a cute animal (not mandatory but encouraged)**
